modules/ROOT/pages/kafka/kafka-connector.adoc

```diff
@@ -1,238 +1,58 @@
-+
-image::kafka/kafka-kerberos-studio-config.png[Kerberos Configuration]
-+
-.. Kerberos SSL:
-** All the parameters from Basic configuration.
-** All the parameters from SSL configuration.
-** All the parameters from Kerberos configuration.
-+
-image::kafka/kafka-kerberos-ssl-studio-config.png[Kerberos SSL Configuration]
-+
-. Based on the operation that you have dragged on the canvas, configure the following fields:
-.. Consumer trigger:
-** Topic - name of Kafka topic to consume messages from.
-** Partition offsets(Optional) - list of Offset representing partitions offsets configuration. For each element in the list you have to specify partition index and offset.
-+
-image::kafka/kafka-consumer-studio-config.png[Consumer Studio Configuration]
-+
-.. Producer operation:
-** Topic - Topic to send the message to
-** Key - Key belonging to the message that is going to be sent
-** Message - Message to be sent
-+
-image::kafka/kafka-producer-studio-config.png[Producer Studio Configuration]
-
-== Use Case: Consuming a Topic
-
-Kafka connector can be used for consuming messages from a particular topic and feeding the flow with those messages, or producing a message into a topic.
-
-In Studio this is what you see:
-
-Consumer:
-
-image::kafka/kafka-consumer-studio-flow.png[Consumer on canvas]
+== Prerequisites
 
-Producer:
+Before creating an app, you must have access to the Kafka target resource and 
+Anypoint Platform. You must also understand how to create a Mule app using 
+Design Center or Anypoint Studio.
 
-image::kafka/kafka-producer-studio-flow.png[Producer on canvas]
+You also need access to Kafka to get values for these fields that appear in Studio and Design Center:
 
-=== Use Case: XML
+[%header,cols="30s,70a"]
+|===
+|Field |Description
+|Group id |A unique string that identifies the consumer group this consumer belongs to.
+|Bootstrap Servers |Comma-separated host-port pairs used for establishing the initial connection to the Kafka cluster. This is the same as the "bootstrap.servers" value you must provide to Kafka clients (producer and consumer). If this property is provided with a producer and consumer properties files, this value is ignored and the one from the properties file is used.
+|Principal |(Kerberos and Kerberos SSL) https://www.google.com/search?q=kerberos+principal[Kerberos principal]
+|Service name |(Kerberos and Kerberos SSL) The Kerberos principal name that Kafka broker runs as.
+|Additional JAAS Properties |(Kerberos and Kerberos SSL) Additional properties as key-value pairs that you need to set in the `sasl.jaas.config` file and that you usually include in Java Authentication and Authorization Service (JAAS) configuration file.
+|Key Store Type |(Kerbero SSL) The file format of the key store file. This is optional for client.
+|Key Store Password |(Kerbero SSL) The store password for the key store file. This is optional for client and only needed if "keyStoreLocation" is configured.
```

Comment:
```suggestion
|Key Store Password |(Kerberos SSL) The store password for the key store file. This is optional for client and only needed if `keyStoreLocation` is configured.
```
I think I saw elsewhere that you changed client to connector. Ignore me if that's not the case.